src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.scss

```diff
@@ -0,0 +1,3 @@
+cds-modal .question > span > p {
+  display: none !important;
+}
```

Comment:
nope, especially with `!important` tag which is discouraged in the inner scss. Instead introduce an extra flag to the delete-modal asking for `showQuestion` which is default true always and make it false for here. you just need to hide the `div class=question` inside that condition.